docs/03-extensions/gezy/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Gezy"
+description:
+  "This extension contains the connector to use Front-Commerce with a headless
+  Gezy backend."
+---
```

Comment:
**Suggestion:** I think it would be good to start using [doc tags](https://docusaurus.io/docs/create-doc#doc-tags)
```suggestion
---
title: "Gezy"
description:
  "This extension contains the connector to use Front-Commerce with a headless
  Gezy backend."
tags:
  - connector
  - gezy
  - headless backend
---
```

Reply:
How can we use them afterwards? Should it go to each page?

I don't understand the value yet.